### PR TITLE
Move tests to CircleCI

### PR DESCRIPTION
**Pull request by Harriethw**
# Context

Trello: https://trello.com/c/HcNQAc05/260-tech-debt-parallelise-ui-tests-on-circleci

Our tests running in Github Actions were starting to hit 12-14 min run time. Moving them over to CircleCI and using CAS1's parallelisation [strategy looks like it shaves over half the time](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui?branch=chore%2Fintegration-tests-circleci). This also has the added benefit that we have the same workflows as CAS1, which will make it easier for the handover team to maintain. 

# Changes in this PR

* We're updating npm/node/wiremock/hmpps orb
* removing a deprecated field in Cypress config
* adding the CirceCI config used by CAS1
* refactoring the wiremock stubbed response for updating an application - this was causing CI tests to fail due to a malformed JSON response. The change brings it in line with the response we expect, and follows the pattern from the other responses.
* removing the GitHub actions we've replaced

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: integration_tests/mockApis/applications.ts [hunk 1]
```diff
@@ -48,18 +49,7 @@ export default {
...
-          "person": ${JSON.stringify(args.application.person)},
-          "createdByProbationOfficerId": "${args.application.createdByUserId}",
-          "schemaVersion": "${args.application.schemaVersion}",
-          "createdAt": "${args.application.createdAt}",
-          "submittedAt": "${args.application.submittedAt}",
-          "data": {{{jsonPath request.body '$.data'}}}
-        }
-        `,
-        transformers: ['response-template'],
+        jsonBody: { ...args.application, type: 'CAS2' },
```
Comment (edavey) [inline integration_tests/mockApis/applications.ts:52]: I'm guessing that `jsonBody` rather than `body` invokes some inbuilt JSON parsing?

**Review by edavey — APPROVED** [review]
🏅 fiddly stuff! Thanks for working all this out.

**Review by patrickjfl — APPROVED** [review]
🙌

File: integration_tests/mockApis/applications.ts (hunk 1 above)
Comment (Harriethw) [reply integration_tests/mockApis/applications.ts:52]: yep exactly. I'll pop a link to the docs in the commit message https://wiremock.org/docs/stubbing/#specifying-the-response-body

**Pull request merged**